tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -202,7 +192,7 @@ module.exports = {
 
       await this.useXpath()
         .moveToElement(this.getFileRowSelectorByFileName(folder), 0, 0)
-        .click(this.getFileLinkSelectorByFileName(folder))
+        .click(filesRow.getFileLinkSelectorByFileName(folder))
```

Comment:
`getFileLinkSelectorByFileName` returns xpath for file-row of  provided file
IMO the function is part of PageObject `fileRow`, what do yo think??

Reply:
`getFileLinkSelectorByFileName` is already in PageObject `filesRow`

Reply:
OK. Then, what about function `getFileRowSelectorByFileName`?

Reply:
I also think that `getFileRowSelectorByFileName` should be in `filesRow`.